packages/apollo-server-lambda/src/ApolloServer.ts

```diff
@@ -116,7 +116,6 @@ export class ApolloServer extends ApolloServerBase {
         const acceptHeader = event.headers['Accept'] || event.headers['accept'];
         if (acceptHeader && acceptHeader.includes('text/html')) {
           const playgroundRenderPageOptions: PlaygroundRenderPageOptions = {
-            endpoint: event.requestContext.path,
```

Comment:
Removing this line means that the graphql endpoint is left as the last user's endpoint, rather than the lambda's graphql endpoint. Do you think this should be the default?

Reply:
I'm going to revert the change, so that we are able to get a release out. If we should have a discussion, I'd love to chat over another PR!

Reply:
@evans I will do some tests to see if it works correctly with custom domains